apps/web/src/contexts/modal/AnimatedModal.tsx

```diff
@@ -81,9 +83,17 @@ function AnimatedModal({
     return 'unset';
   }, [isFullPage, isMobile]);
 
+  const onClickHandler = () => {
+    if (onBypassClose) {
+      onBypassClose();
+    } else {
+      hideModal();
+    }
+  };
```

Comment:
1. wrap in `useCallback`
2. could we rename this function from `onClickHandler` -> `handleClick`